lib/getChats.ts

```diff
@@ -0,0 +1,6 @@
+import backendFetch from './backendFetch';
+import Chat from '../types/Chat';
+
+export default async function getChats(): Promise<Chat[] | null> {
+  return await backendFetch<Chat[] | null>('GET', 'chat/'); // returns Chats
```

Comment:
Does the chat endpoint ever return `null`? I don't see a scenario where it would
```suggestion
  return await backendFetch<Chat[]>('GET', 'chat/'); // returns Chats
```